docs/04_web/02_web-components/01_form/04_file-upload.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Web Components - File Upload'
+sidebar_label: 'File Upload'
+id: file-upload
+keywords: [web, web components, file upload]
+tags:
+    - web
+    - web components
+    - file upload
+---
+
+The `file-upload` component can be used to select single or multiple files from the local file system. Selected files can be uploaded to a server using the **Upload** button.
+
+![](/img/file-upload-component-file-selected.PNG)
+
+:::note 
+The Upload button is disabled if no file is selected.
+::: 
+
+In addition, the uploaded files can be viewed in the [`data-grid`](../../grids/data-grid/). The `data-grid` also provides a **Download** button.
+
+![](/img/file-upload-component.PNG)
+
+## Set-up
+In order to use the `file-upload` component, you need to import it from a design system. In the example below, we import it from zero-design-system and register it.
+
+```ts
+import { provideDesignSystem, zeroFileUpload } from '@genesislcap/zero-design-system';
+
+provideDesignSystem().register(zeroFileUpload());
+```
+
+## Usage
+The `file-upload` component can be used as shown below:
+
+```html title="Basic example"
+    <zero-file-upload     
+          entity-id="Demo"
+          field-name="ENTITY_ID"
+          upload-key="series">
+    </zero-file-upload>
+```
+:::note

```

Comment:
```suggestion
:::warning

```

Reply:
I think its a bit misleading because there is no immediate danger on missing the attribute.
![image](https://github.com/genesiscommunitysuccess/docs/assets/118179643/8e9458c0-f3b8-4c64-8a20-598e34cc0268)
But I do agree with you that `note` is not enough. what do you think if we use `important` instead?
![image](https://github.com/genesiscommunitysuccess/docs/assets/118179643/f91ccd1b-bb90-483e-92b4-d38a93fb1ac8)

Reply:
ok will change it to warning.